docker-compose/forums/docker-compose.yml

```diff
@@ -0,0 +1,35 @@
+db:
+  container_name: db
+  image: edxops/mysql:v1
+  environment:
+    - MYSQL_ROOT_PASSWORD=''
+  hostname: db
+  ports:
+    - 3306:3306
+
+mongo:
+  container_name: mongo
+  image: edxops/mongo:v2
+  volumes: 
+    - ${DOCKER_DATA_ROOT}/mongo/data:/data
+
+es:
+  container_name: es
+  image: edxops/elasticsearch:v1
+  volumes:
+    - ${DOCKER_DATA_ROOT}/elasticsearch/data:/es-data
+
+memcache:
+  container_name: memcache
+  image: memcached:1.4.24
+
+forums:
+  container_name: forums
+  image: edxops/forums:opencraft
+  ports:
+    - 4567:4567
```

Comment:
It might be good to mount the local forum checkout at the appropriate point in the volume.